data-es7.js

```diff
@@ -738,7 +763,10 @@ exports.tests = [
   category: 'strawman',
   link : 'https://github.com/DavidBruant/Map-Set.prototype.toJSON',
   exec: function(){/*
-    return JSON.stringify(new Map([['a', 'b'], ['c', 'd']])) === '[["a","b"],["c","d"]]';
+    var map = new Map();
+    map.set('a', 'b');
+    map.set('c', 'd');
+    return JSON.stringify(map) === '[["a","b"],["c","d"]]';
```

Comment:
You really think any browser will add `{Map, Set}#toJSON` w/o support initialization from iterable? :) Even such test will not work in IE9. Simply add your `es6-shim` as test dependency for `es7-shim`.